features/src/main/feature/camel-features.xml

```diff
@@ -1357,6 +1357,20 @@
         <bundle>mvn:org.apache.camel.karaf/camel-attachments/${project.version}</bundle>
         <bundle>mvn:org.apache.camel.karaf/camel-mail/${project.version}</bundle>
     </feature>
+    <feature name='camel-mail-microsoft-oauth' version='${project.version}' start-level='50'>
+        <feature version='${camel.osgi.version.range}'>camel-mail</feature>
+        <feature version='[2.16,2.17)'>jackson</feature>
+        <bundle dependency='true'>wrap:mvn:com.microsoft.azure/msal4j/${msal4j-version}$overwrite=merge&amp;Import-Package=com.sun.net.httpserver;resolution:=optional,*</bundle>
+        <bundle dependency='true'>mvn:com.nimbusds/content-type/2.3</bundle>
+        <bundle dependency='true'>mvn:com.nimbusds/nimbus-jose-jwt/${nimbus-jose-jwt}</bundle>
+        <bundle dependency='true'>wrap:mvn:com.github.stephenc.jcip/jcip-annotations/1.0-1</bundle>
+        <bundle dependency='true'>mvn:com.nimbusds/oauth2-oidc-sdk/11.9.1</bundle>
+        <bundle dependency='true'>mvn:com.nimbusds/lang-tag/1.7</bundle>
+        <bundle dependency='true'>mvn:net.minidev/json-smart/${json-smart-version}</bundle>
```

Comment:
`oauth2-oidc-sdk-11.9.1.jar` manifest requires minidev dependencies to be at least of a version 2.5

```
  net.minidev.json;version="[2.5,3)",net.m
 inidev.json.parser;version="[2.5,3)",net.minidev.json.writer;version="[
 2.5,3)"

 Dependency tree in camel project suggests
[INFO]    +- net.minidev:json-smart:jar:2.4.10:compile
[INFO]    |  \- net.minidev:accessors-smart:jar:2.4.9:compile
[INFO]    |     \- org.ow2.asm:asm:jar:9.3:compile
```

However, we use `<json-smart-version>2.5.1</json-smart-version>` in already existing features and that complies with `oauth2-oidc-sdk-11.9.1.jar` manifest, so used this version to avoid wrap of this bundle to override